nuxeo-features/nuxeo-platform-oauth/src/main/java/org/nuxeo/ecm/platform/oauth2/tokens/NuxeoOAuth2TokenWriter.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Contributors:
+ *     Gabriel Barata <gbarata@nuxeo.com>
+ */
+package org.nuxeo.ecm.platform.oauth2.tokens;
+
+import static org.nuxeo.ecm.core.io.registry.reflect.Instantiations.SINGLETON;
+import static org.nuxeo.ecm.core.io.registry.reflect.Priorities.REFERENCE;
+
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+
+import org.codehaus.jackson.JsonGenerator;
+import org.nuxeo.ecm.core.io.marshallers.json.ExtensibleEntityJsonWriter;
+import org.nuxeo.ecm.core.io.registry.reflect.Setup;
+
+/**
+ * @since 9.2
+ */
+@Setup(mode = SINGLETON, priority = REFERENCE)
+public class NuxeoOAuth2TokenWriter extends ExtensibleEntityJsonWriter<NuxeoOAuth2Token> {
+
+    public static final String ENTITY_TYPE = "nuxeoOAuth2Token";
+
+    public static final String DATE_FORMAT = "yyyy-MM-dd";
```

Comment:
Don't we want a finer format, to the second at least? I'm don't know if this is part of any spec. In the test CSV you have a time to the seconds...